documentation/source/users/rmg/input.rst

```diff
@@ -7,15 +7,15 @@ Creating Input Files
 
 The syntax and parameters within an RMG input file are explained below.  We recommend
 trying to build your first input file while referencing one of the
-:ref:`Example Input Files<examples>`.  Alternatively, you can use our web form found 
```

Comment:
This PR gets rid of a lot of whitespace issues (which I like), but it makes the history hard to read because of it. Any way we can get around this?

Reply:
I feel like if there's one thing we should sacrifice history for to improve readability it's probably the documentation.